plugin/core/transports.py

```diff
@@ -300,7 +301,7 @@ def start_subprocess() -> subprocess.Popen:
             reader = process.stdout
             writer = process.stdin
     stderr_reader = process.stdout if config.node_ipc else process.stderr
```

Comment:
Can you point me to some docs on why for multiprocessing.connection the process.stdout is used instead of process.stderr?

Reply:
I redirect stdout to stderr here:

https://github.com/sublimelsp/LSP/pull/2015/files#diff-2abf74ab722813d8b12a5da0a054ac7ba7ee95f47278607834d5dc672357b818R265

To have just one pipe to monitor, instead of two. Can't do the opposite redirection, unfortunately.